src/%IPM/Main.cls

```diff
@@ -581,6 +581,25 @@ generate /my/path -export 00000,PacketName2,IgnorePacket2^00000,PacketName3,Igno
 <parameter name="name" description="Name namespace, * or context name*" />
 </command>
 
+	<command name="enable">
+		<summary>
+			Enable IPM in other namespaces.
+		</summary>
+		<!-- Modifiers -->
+		<modifier name="version" aliases="v" value="true" description="Specific version of IPM to be installed. Must specify if quiet flag is set."/>
+		<modifier name="namespaces" aliases="ns" value="true" description="Comma-separated namespaces in which IPM needs to be enabled."/>
+		<modifier name="globally" description="Will install IPM in all non-%SYS explicit namespaces that currently do not have IPM installed."/>
```

Comment:
Since we have this globally enable option, please create corresponding jiras for new commands `disable` and and `ls` (to list IPM versions across namespaces)